packages/api-headless-cms/__tests__/multipleValues.test.js

```diff
@@ -121,4 +121,31 @@ describe("Multiple Values Test", () => {
             `Cannot change "multipleValues" for the "tags" field because it's already in use in created content.`
         );
     });
+
+    it(`should not be able to set "multipleValues" as index`, async () => {
```

Comment:
`should not be able to set "multipleValues" as index` => `should prevent the creation of an index that includes a multiple values field in the fields list`